app/models.py

```diff
@@ -132,6 +139,32 @@ class Service(db.Model, Versioned):
     )
 
 
+class ServiceWhitelist(db.Model):
+    __tablename__ = 'service_whitelist'
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), index=True, nullable=False)
+    service = db.relationship('Service', backref='whitelist')
+    email_address = db.Column(db.String(255), nullable=True)
+    mobile_number = db.Column(db.String, nullable=True)
```

Comment:
Is the intention of the whitelist to have both an email and mobile number or one or the other?